### ISSUE TITLE
RTL Crash right after cold start - System.ArgumentNullException: Value cannot be null. Parameter name: items in ReactiveSectionedRecyclerAdapter

### ISSUE THREAD

**Issue by semanticer** [toggl/mobileapp#5989]
## 💥 Crash Report
This happed every time I changed the system language to some of the RTL and cold started the app. You can also logout, change to RTL and that try to login = same result.

### 👥 Impact
This is crash 
So far, this only happened to me during 2.4 testing on my OG Pixel with Android 10 on my personal production account.

### 🔗 Context
We had similar crash in 2.0.1 https://appcenter.ms/orgs/Toggl/apps/Toggl-Android/crashes/errors/1819654063u/overview 

### 🛰 Link
https://appcenter.ms/orgs/Toggl/apps/Toggl-Android/crashes/errors/3897953320u/overview

### 📝  Exception & Stack Trace
`Exception type and message`
```cs
Requires.FailArgumentNullException (System.String parameterName)
Requires.NotNull[T] (T value, System.String parameterName)
ImmutableList`1[T].AddRange (System.Collections.Generic.IEnumerable`1[T] items)
ImmutableList.ToImmutableList[TSource] (System.Collections.Generic.IEnumerable`1[T] source)
ReactiveSectionedRecyclerAdapter`7[TModel,TModelWrapper,TSectionHeaderModel,TSectionHeaderWrapper,TItemViewHolder,TSectionViewHolder,TKey].UpdateCollection (System.Collections.Generic.IEnumerable`1[T] items)
AnonymousSafeObserver`1[T].OnNext (T value)
Sink`1[TTarget].ForwardOnNext (TTarget value)
ObserveOnObserverNew`1[T].DrainStep (System.Collections.Concurrent.ConcurrentQueue`1[T] q, System.Boolean delayError)
ObserveOnObserverNew`1[T].DrainShortRunning (System.Reactive.Concurrency.IScheduler recursiveScheduler)
<.cctor>b__20_1 (System.Reactive.Concurrency.IScheduler scheduler, System.Reactive.ObserveOnObserverNew`1[T] self)
HandlerScheduler+<>c__DisplayClass6_0`1[TState].<Schedule>b__0 ()
Thread+RunnableImplementor.Run ()
IRunnableInvoker.n_Run (System.IntPtr jnienv, System.IntPtr native__this)
```

**Comment by semanticer** [comment]
This is interesting. It is crashing reliably for me when I use the production version with RTL language even when I'm logged out or installing the app for the first time from Google Play.  
BUT it's impossible to reproduce on dev build even with changing API endpoint to production.

**Comment by semanticer** [comment]
To add more info - it looks like it's happening reliably only with Afganistan locale and it turns out it's happening on 2.3 version as well.  

Afganistan's language looks like this https://photos.app.goo.gl/PRskvztMeDH4vQuo7

**Comment by daividssilverio** [comment]
@semanticer , from my tests, it seems like this issue has been collaterally solved by #6083. Could you confirm as I haven't been able to replicate this crash?

**Comment by semanticer** [comment]
Yes, confirmed 👍 after years of being a victim of collateral damage, Afganistan was finally target of some collateral fix as well 🎉
